src/entry-io.js

```diff
@@ -70,15 +71,16 @@ class EntryIO {
     }
 
     // Add entries that we don't need to fetch to the "cache"
-    const addToExcludeCache = e => { cache[e.hash] = true }
+    const addToExcludeCache = e => { cache[e.hash || e] = true }
```

Comment:
`exclude` is an array of hashes. The change here was to fix it when we added hashes from `entry.next` or `entry.refs` since the element `e` being added wasn't an `Entry` but a hash.